.github/workflows/tests.yml

```diff
@@ -135,7 +135,7 @@ jobs:
     runs-on: [self-hosted, Linux, large, ephemeral]
     steps:
       - name: Download Test Reports
-        uses: actions/download-artifact@65a9edc5881444af0b9093a5e628f2fe47ea3b2e # v4.1.7
```

Comment:
you don't mention why you are downgrading `actions/download-artifact` to an older version in the description of this PR

Reply:
Updated description

Reply:
thanks!